### ISSUE TITLE
Generated bundle is causing issues.

### ISSUE THREAD

**Issue by svnscha** [NetCoreTemplates/react-lite#2]
I've discovered another issue.

```
bundle.js:74 Uncaught (in promise) RangeError: Maximum call stack size exceeded
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
    at Object.get [as Hello] (bundle.js:710)
```

just after creating the project I get this error message in the Developer Tools. Then I tried to re-generate the DTO bindings I ran into this:

```
xyz@xyz ProjectName % x ts
Could not update ServiceStack Reference '/Users/xyz/Development/ProjectName/src/shared/dtos.ts': 
One or more errors occurred. (Connection refused)
```

**Comment by mythz** [comment]
Can you provide the exact steps to repro please? I'm able to run `_bundle.ss` without issue:

```
C:\projects\ReactLite>x run wwwroot\_bundle.ss
<link rel="stylesheet" href="/css/lib.bundle.1588388884000.min.css">
<script src="/js/lib.bundle.1588388884000.min.js"></script>
<script src="/js/bundle.1589620884872.min.js"></script>
```

**Comment by mythz** [comment]
Actually the issue with `dtos.ts` is that it's looking at the old URL, it should be:

```
BaseUrl: https://localhost:5001
```

I'll update the project, please provide repro steps for the first issue.

**Comment by svnscha** [comment]
Changing the URL doesn't solve the issue. I tried that manually already.

Invoking the command `x ts` in project root directory outputs:

```
Could not update ServiceStack Reference '/Users/elliot/Development/ProjectName/src/shared/dtos.ts': 
One or more errors occurred. (Connection refused)
```

The first issue is even more simple to reproduce, create a new project with

1. `x new react-lite ProjectName`
2. `cd ProjectName`
3. `tsc -w`
4. `dotnet run`
5. Open the website and open the Chrome developer tools. The issue is then shown.

**Comment by mythz** [comment]
> One or more errors occurred. (Connection refused)

Suggests the project isn't running at the https://localhost:5001 which the `BaseUrl` points to:

I'm not able to repro it with those steps:

![image](https://user-images.githubusercontent.com/89361/82117910-2cc9a800-97a6-11ea-880d-e56b6bbf731c.png)

**Comment by svnscha** [comment]
![image](https://user-images.githubusercontent.com/24257042/82117968-3fc38480-9774-11ea-9b10-22c878378c68.png)

It indeed is but probably it is a certificate thing.

**Comment by mythz** [comment]
Yeah looks like you don't have a local trusted self-signed cert, it should automatically configured with .NET Core install, but maybe this blog post from Scott Hanselman [Developing locally with ASP.NET Core under HTTPS, SSL, and Self-Signed Certs](https://www.hanselman.com/blog/DevelopingLocallyWithASPNETCoreUnderHTTPSSSLAndSelfSignedCerts.aspx) can provide tips on how to configure a local self-signed cert.

**Comment by svnscha** [comment]
I fixed this by cleaning and re-importing the localhost certificate. That was weird in my setup.

https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-2.2&tabs=visual-studio#trust-the-aspnet-core-https-development-certificate-on-windows-and-macos

by just doing 

```
dotnet dev-certs https --clean
dotnet dev-certs https --trust
```

However, the issue still persists. It outputs once page-load and everytime you enter something in the Input field.

![image](https://user-images.githubusercontent.com/24257042/82118120-5dddb480-9775-11ea-9d5c-122536740268.png)

**Comment by mythz** [comment]
Yeah unfortunately I'm unable to repro this issue, so can't say what the problem is. Can you check the network inspector to make sure there aren't any >=400 errors with any of the requests also check the bundle.js request to check if it's returning the App's js bundle.
